enterprise-suite/scripts/lbc.py

```diff
@@ -464,6 +464,21 @@ def install(creds_file):
 
     helm_args = ''
     if len(args.helm) > 0:
+        # namespace can be passed to helm directly after -- and can take format -namespace <value> or --namespace=<value>
+        # remove namespace from helm args and assign it args.namespace
+        # handle --namespace <val>
+        if  "--namespace" in args.helm and args.helm.index("--namespace") < len(args.helm):
+            ns_index=args.helm.index("--namespace")
+            args.namespace=args.helm[ns_index+1]
+            del args.helm[ns_index:ns_index+2]
+
+        # handle --namspace=<val>
+        r=re.compile("--namespace=.*")
+        if filter(r.match, args.helm):
+            ns_val=filter(r.match, args.helm)[0]
+            args.namespace=ns_val.split("=")[1]
+            del(args.helm[args.helm.index(ns_val)])
```

Comment:
Is this to handle the situation where a user does `--namespace=a -- --namespace=b`? If so, I think it would be better to log an error and exit.